Frontend/CO.CDP.OrganisationApp/Models/DynamicForms.cs

```diff
@@ -78,5 +79,6 @@ public enum FormQuestionType
     Date,
     CheckBox,
     Address,
-    MultiLine
+    MultiLine,
+    GroupedSingleChoice
```

Comment:
Do we also need to update the data sharing response with the new type?

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/3ed146f0b21806f47aea81fdbdfff45affbbf779/Services/CO.CDP.DataSharing.WebApi/AutoMapper/DataSharingProfile.cs#L122-L169

Reply:
@jakzal Hi Kuba, I think yes we need to return because it will hold at least 5 lines of text messages, i.e. more characters than a normal text box. 
And now we will have Single choice and the GroupedSingleChoice; they will both hold one value.